src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
 
   }
+  if (a == null)
+   a.charAt(123);
```

Comment:
**Reporter**: FINDBUGS
**Rule**: NP_ALWAYS_NULL
**Severity**: ERROR
**File**: src/main/java/se/bjurr/violations/lib/example/MyClass.java L13
**Source**: se.bjurr.violations.lib.example.MyClass

Null pointer dereference


<p> A null pointer is dereferenced here.&nbsp; This will lead to a
<code>NullPointerException</code> when the code is executed.</p>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1116723583>*